rules/technical-overview/rule.md

```diff
@@ -0,0 +1,47 @@
+---
+type: rule
+title: Rules to having a good technical overview
+uri: technical-overview
+authors:
+  - title: Bryden Oliver
+    url: https://www.ssw.com.au/people/bryden-oliver
+  - title: Ulysses Maclaren
+    url: https://www.ssw.com.au/people/uly
+  - title: Patrick Zhao
+    url: https://www.ssw.com.au/people/patrick-zhao
+  - title: Adam Cogan
+    url: https://www.ssw.com.au/people/adam-cogan
+created: 2022-04-21T23:39:49.013Z
+guid: e9a156f4-f518-4ea9-924a-42811ee10e48
+---
+When running a project, having someone with a wide overview is extremely valuable. That person is much more likely to notice broader problems and be able to act to correct them. 
+It is highly effective to have this person attend sprint reviews and planning. Attending daily scrums is also valuable but not essential, even attending occasionally has value. 
+
+This person's objectives should include the following:
+1. Provide feedback to the Product Owner on what’s in backlog
+    - Are items sensibly sized?

```

Comment:
Make clear that "sensibly sized" is about breaking down tasks, not critiquing developer estimations